backend/handlers/login.js

```diff
@@ -1,60 +0,0 @@
-const user = require('../../database/userhelpers.js');
-const Boom = require('boom');
-const Bcrypt = require('bcrypt');
-module.exports = (req, res) => {
-  const email = req.payload.email;
-  const password = req.payload.password;
-  user.getuserbyemail(email, (error, result) => {
-    if (error) {
-      {
-          // eslint-disable-next-line no-console
-        console.log('get user by email Error :',error)
-        return res().code(500)
-      }
-    }
-    if (result.rows.length > 0) {
-      Bcrypt.compare(password, result.rows[0].password, (error, resultHash) => {
-        if (error) {
-          // eslint-disable-next-line no-console
-          console.log('Bcrypt comapre Error  :',error)
-          return res().code(500)
-
-        }
-        if (resultHash) {
-          if (result.rows[0].approved === 0) {
-            res(Boom.forbidden('Wait until the admin approved your request'))
-          } else {
-            req.cookieAuth.set({
-              user_id: result.rows[0].user_id,
```

Comment:
again, result.rows[0] keeps being reused. either define as a variable within this module or at the "result" object level.